modules/api/src/main/java/org/apache/ignite/lang/ColumnNotFoundException.java

```diff
@@ -0,0 +1,15 @@
+package org.apache.ignite.lang;
+
+/**
+ * Exception is thrown when appropriate column is not found.
+ */
+public class ColumnNotFoundException extends IgniteException {
+    /**
+     * Create a new exception with given column name.
+     *
+     * @param name Column name.
+     */
+    public ColumnNotFoundException(String name) {
```

Comment:
The only column name looks not informative.
Is it possible to add more context here? schema and table name?

Reply:
Changed for: Column '{}' does not exist in table '{}' is it ok ?